src/components/Integrations/Integrations.tsx

```diff
@@ -0,0 +1,93 @@
+'use client';
+
+import React from 'react';
+import './Integrations.css';
+import { useWebhooks } from '@/hooks/useWebhooks';
+import { WebhookForm } from './WebhookForm';
+import { WebhookTable } from './WebhookTable';
+import { DeleteConfirmModal } from './DeleteConfirmModal';
+
+export const IntegrationsPage = () => {
+    const {
+        webhooks,
+        currentWebhook,
+        setCurrentWebhook,
+        parametersInput,
+        setParametersInput,
+        conditions,
+        setConditions,
+        logic,
+        setLogic,
+        signalNames,
+        generatedCEL,
+        expandedWebhook,
+        setExpandedWebhook,
+        showDeleteConfirm,
+        setShowDeleteConfirm,
+        webhookToDelete,
+        setWebhookToDelete,
+        handleCreate,
+        handleUpdate,
+        handleDelete,
+        resetForm,
+    } = useWebhooks();
+
+    return (
+        <div className="integrations-container">
+            <div className="integrations-header">
+                <h1 className="integrations-title">Webhooks</h1>
+                <button
+                    className="create-webhook-button"
+                    onClick={() => {
+                        resetForm();
+                        setCurrentWebhook({
+                            description: 'My Cool Webhook',
+                            target_uri: 'https://example.com/webhook',
+                        });
+                    }}
```

Comment:
Let's create a function to handle the webhook creation
```suggestion
                    onClick={handleCreate}
```